state-chain/pallets/cf-broadcast/src/migrations/v2.rs

```diff
@@ -2,47 +2,168 @@ use crate::*;
 #[cfg(feature = "try-runtime")]
 use frame_support::sp_runtime::DispatchError;
 use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
-use sp_std::marker::PhantomData;
 #[cfg(feature = "try-runtime")]
 use sp_std::prelude::Vec;
+use sp_std::{collections::btree_map::BTreeMap, marker::PhantomData};
 
 mod old {
-
 	use super::*;
 
+	/// A unique id for each broadcast attempt
```

Comment:
These new migrations need to be in a v3 file, and the appropriate pallet version bump. v2 has already been released.

Reply:
Removed  old migration, advanced version from v2 -> v3